pkg/controller/pv.go

```diff
@@ -114,7 +113,7 @@ func (r *ReconcilePV) Reconcile(ctx context.Context, request reconcile.Request)
 
 		}
 		// AzVolume does exist and needs to be deleted
-		// add annotation to mark AzVolumeAttachment cleanup
+		// add annotation to mark AzVolume cleanup
```

Comment:
This annotation is used for `AzVolumeAttachment` clean up when `AzVolume` for pre-provisioned volume is marked for deletion. So let's leave it as AzVolumeAttachment.

Reply:
Done!